awswrangler/dynamodb/_write.py

```diff
@@ -181,7 +181,7 @@ def put_df(
     ...     table_name='table'
```

Comment:
Same for `_put_df` and `_put_items` so that the user can see what each Ray worker/thread is doing? It would log the number of items for example?

Reply:
Yap `_put_items` is logging how many items it is inserting. And `_put_df` is using `put_items` under the hood.